recipes/grid/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -eu -o pipefail
+
+outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+mkdir -p $outdir
+mkdir -p $PREFIX/bin
+
+cd $SRC_DIR
+
+cp -R * $outdir
```

Comment:
Do we need the test data? The package is big and I wonder if everything is need what you are copying here.

Reply:
It's not necessary but it's nice to have in the installation in order to test it. I can remove it.